README.md

```diff
@@ -35,6 +35,22 @@ should submit the patch on their own behalf.
 
 [The full list of individuals is available here.](individuals.md)
 
+## Howto
+
+Issue a pull request:
+
+- include an edit to the /individuals.md file (put your name in alphabetic
```

Comment:
quoting a file name like you're doing can be enclosed with backticks: ` so that it's emphasized with PRE tags properly =D
Please do this here and below in the other cases.